front/src/components/UI/StarRating/index.jsx

```diff
@@ -0,0 +1,3 @@
+import StarRating from './StarRating';
+
+export default StarRating;
```

Comment:
лучше сразу экспортировать из индекс файла, иначе это двойная работа
 
```suggestion
export {StarRating} from './StarRating'
```

Reply:
Здесь это не работает) Я так писала, была ошибка. Надо удалить arbnb плагин. Я поправлю потом

Reply:
если мы начинаем работать с индекс-файлами для экспортов, то нужно обсудить с командой и создать issue, чтобы все компоненты привести к единому виду